.editorconfig

```diff
@@ -1,14 +1,14 @@
 root = true
 
-[{*.kt,*.kts}]
+[*]
 charset = utf-8
 end_of_line = lf
-indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
-ij_kotlin_allow_trailing_comma = true
-ij_kotlin_allow_trailing_comma_on_call_site = true
-ij_kotlin_packages_to_use_import_on_demand = unset
-ij_kotlin_name_count_to_use_star_import = 2147483647
-ij_kotlin_name_count_to_use_star_import_for_members = 2147483647
```

Comment:
Perhaps controversially I have not retained the 'no-wildcard-imports' config. I don't think the style-guide has anything to say on these, and the IJ default is to allow them, so they are allowed. 

I personally like wildcard imports, despite originally being against them. Perhaps this is because Kotlin generally handles imports and type-inference more stably than Java, so it's less of an issue? Or because Kotlin libraries often have extension functions that are much more convenient to use when wildcard-importing?

In any case, are you okay with this decision?

Reply:
I personally like to know where my imports come from, but I won't oppose here - it's a minor issue to me :)